test/expressions.ml

```diff
@@ -111,6 +111,21 @@ let%expect_test "Functions" =
     │ null ┆ egg   ┆ 0.156988 ┆ C      │
     │ 5    ┆ spam  ┆ 0.831802 ┆ B      │
     └──────┴───────┴──────────┴────────┘ |}];
+  let min_max = Data_frame.select_exn df ~exprs:
```

Comment:
Sorry, the tests don't make this super clear but the tests I see you've changed unfortunately are to the ones that are supposed to correspond to the Polars user guide (see https://github.com/mt-caret/polars-ocaml/pull/28 where we're going to extract them out into a separate directory).

Could we put these expect tests into a new file? I think eventually they belong in doctests in the .mli but that PR (https://github.com/mt-caret/polars-ocaml/pull/37) is blocked on a fairly large number of other PRs.

Reply:
I moved it to a separate file, "expression_test.ml". The name is almost a duplicate now, but that won't be a problem after the other tests are in a separate directory.